package.json

```diff
@@ -1,27 +0,0 @@
-{
```

Comment:
I was worried it'd confuse firefox about the nature of the extension, but that's nonsense and it's back now, sorry.